pybossa/forms/dynamic_forms.py

```diff
@@ -17,3 +20,27 @@ def build_field(field_type, description, validators, kwargs=None):
     kwargs = kwargs or {}
     field_class = getattr(wtforms, field_type)
     return field_class(lazy_gettext(description), **kwargs)
+
+def dynamic_project_form(class_type, form_data, data_access_levels, obj=None):
+
+    class ProjectFormExtraInputs(class_type):
+        pass
+
+    if data_access_levels:
+        data_access = Select2Field(
+            lazy_gettext('Access Level(s)'),
+            [validators.Required()],
+            choices=data_access_levels['valid_access_levels'],
+            default=[])
+        setattr(ProjectFormExtraInputs, 'data_access', data_access)
+
+
+    product = SelectField(lazy_gettext('Product'),
```

Comment:
Can these be added to the base class?

Reply:
yes I thinks is there I forgot to remove it from here.